### ISSUE TITLE
Table output

### ISSUE THREAD

**Issue by Soullivaneuh** [Soullivaneuh/composer-versions-check#22]
Maybe a table output will be better:

Package | Installed | Latest | Depends
------------ | ------------- | ------------ | -------------
symfony/symfony | 2.3.30 | 2.7.5 | ~
foo/bar | 1.0.4 | 2.0.5 | awesome/package (~1.0)

**Comment by jakoch** [comment]
Hey Sullivan! 

Would it be possible to generate the following `bower list` like output instead of a table?
![Bower list](http://i.stack.imgur.com/J4krU.png)
I think a renderer base class using adapters could make sense.
You could then implement a TableRenderer (using Symfony\Console Table helper) and a TreeRenderer.

What do you think?

**Comment by Soullivaneuh** [comment]
Could be a second option indeed but we have to be sure the output is not confusing by specified that children tree is dependencies.

Do you have some suggestions of library to render this kind of output easily?

**Comment by jakoch** [comment]
Indeed, listing the dependencies could be confusing, especially if you render a really large set of dependencies.

A library, hmm lets see.. this could fit: https://github.com/pbergman/tree-printer
It stays in the context of Symfony/Console.
I haven't worked with it, but it allows to define the "ascii art" forming the tree.
And it should be possible to tweak the settings to render a more compact tree.
https://github.com/pbergman/tree-printer#setformats

**Comment by stephanvierkant** [comment]
Here is a screenshot from VersionEye.com:

![selection_170](https://cloud.githubusercontent.com/assets/601833/21216677/7587c7d8-c2a8-11e6-9899-08b7dc51c411.png)

I really like this view. An output like `bower list` can be very confusing. I think this package should show version numbers, not a dependencies tree.
